Examples/Algorithms/Generators/ActsExamples/Generators/ParametricParticleGenerator.cpp

```diff
@@ -100,6 +101,7 @@ ParametricParticleGenerator::operator()(RandomEngine& rng) {
     // construct the particle;
     ActsFatras::Particle particle(pid, pdg, q, m_mass);
     particle.setDirection(dir);
+    p = m_cfg.pLogUniform ? std::exp(p) : p;
```

Comment:
Similarly here. I don't see a perfect alternative. Maybe, we move that line to 
```
l.82    double p = pDist(rng);
```
to have it closer to the drawing?